qt_ui/widgets/spinsliders.py

```diff
@@ -27,28 +28,29 @@ def value(self) -> float:
 
 
 class TimeInputs(QtWidgets.QGridLayout):
-    def __init__(self, label: str, initial: int) -> None:
+    def __init__(self, label: str, initial: timedelta) -> None:
         super().__init__()
         self.addWidget(QtWidgets.QLabel(label), 0, 0)
 
-        minimum = 30
-        maximum = 150
+        minimum_minutes = 30
+        maximum_minutes = 150
+        initial_minutes = int(initial.total_seconds() / 60)
```

Comment:
This is fine too so non-blocking, but FYI you can use `a // b` for integer division.